react/__mocks__/vtex.dreamstore-icons.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const IconCart = props => <div> Icon Cart {props.children} </div>
```

Comment:
could you add a `class` for each one? ie: `className="iconcart-mock"`

or you could mock like [this](https://github.com/vtex-apps/product-kit/blob/master/react/__mocks__/vtex.dreamstore-icons.js):